godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/type/CoreType.kt

```diff
@@ -0,0 +1,21 @@
+package godot.core
+
+import kotlinx.cinterop.*
+
+
+internal interface CoreType {
+    fun getRawMemory(memScope: MemScope): COpaquePointer
+    fun setRawMemory(mem: COpaquePointer)
+}
+
+fun Long.getRawMemory(memScope: MemScope): COpaquePointer {
```

Comment:
I think we need to merge our way to do, [this one](https://github.com/utopia-rise/godot-kotlin/blob/feature/api-code-generation-as-gradletask/godot-kotlin/godot-library/src/nativeInternal/kotlin/godot/internal/utils/getCValue.kt) has been added to api generation branch.
In any case, it should be internal.